cms/templates/ux/reference/modal_access-component.html

```diff
@@ -0,0 +1,126 @@
+                          </li>
+
+                          <li class="field field-checkbox field-visibility-content-group">
+                            <input type="checkbox" id="visibility-content-group-NAME4" name="visibility-content-group" value="" class="input input-checkbox visibility-content-group-NAME4" />
+                            <label for="visibility-content-group-NAME4" class="label">Content Group NAME 4</label>
+                          </li>
+
+                          <!-- NOTE: @andyarmstrong, here's an example of how we would treat a group that was deleted/removed - we need a .was removed class and an additional UI element called a .note -->
+                          <li class="field field-checkbox field-visibility-content-group was-removed">
+                            <input type="checkbox" id="visibility-content-group-deleted1" name="visibility-content-group" value="" class="input input-checkbox visibility-content-group-deleted" checked="checked" />
+                            <label for="visibility-content-group-deleted1" class="label">
+                              Deleted Content Group
+                            </label>
+                            <span class="note">The selected group no longer exists. Choose another group or make the component visible to All Students and Staff</span>
+                          </li>
+
+                          <!-- NOTE: @andyarmstrong, here's an example of how we would treat a group that was deleted/removed - we need a .was removed class and an additional UI element called a .note -->
+                          <li class="field field-checkbox field-visibility-content-group was-removed">
+                            <input type="checkbox" id="visibility-content-group-deleted1" name="visibility-content-group" value="" class="input input-checkbox visibility-content-group-deleted" checked="checked" />
+                            <label for="visibility-content-group-deleted1" class="label">
+                              Deleted Content Group
+                            </label>
+                            <span class="note">The selected group no longer exists. Choose another group or make the component visible to All Students and Staff</span>
+                          </li>
+                        </ul>
+                      </div>
+                    </section>
+                  </div>
+                </div>
+              </div>
+
+            </form>
+
+          </div><!-- .xblock -->
+        </div><!-- .xblock-editor -->
+      </div><!-- .modal-content -->
+
+      <div class="modal-actions">
+        <h3 class="sr">Actions</h3>
+        <ul>
+          <li class="action-item">
+            <a href="#" class="button action-primary action-save">Save</a>
+          </li>
+
+          <li class="action-item">
+            <a href="#" class="button  action-cancel">Cancel</a>
```

Comment:
As I mentioned above, ESC should perform the same action.  Click either button should return focus to the eye/visibility icon.  tabbing past this point should move focus back to the beginning of the dialog box (trapping focus), just as shift tabbing from the first focusable element should return focus here.